avaframe/out3Plot/outDebugPlots.py

```diff
@@ -2,7 +2,7 @@
 import pathlib
 import copy
 import matplotlib
-matplotlib.use('agg')
+# matplotlib.use('agg')
```

Comment:
Please leave this in -> needed for operational run

Reply:
how do we deal with  this because I want to see the plots but with this I cant

Reply:
I'll convert this to an issue. We need to find a automatic solution dependent on what's available in the local environement. Problem originates because QGis often has limited python envs...